packages/scene-composer/src/components/panels/SettingsPanel.tsx

```diff
@@ -67,51 +69,63 @@ export const SettingsPanel: React.FC<SettingsPanelProps> = ({ valueDataBindingPr
   return (
     <Fragment>
       <ExpandableInfoSection
-        title={intl.formatMessage({ description: 'ExpandableInfoSection Title', defaultMessage: 'Scene Settings' })}
+        title={intl.formatMessage({
+          description: 'ExpandableInfoSection Title',
+          defaultMessage: 'Current View Settings',
+        })}
         defaultExpanded
       >
-        <SpaceBetween size='s'>
-          <FormField
-            label={intl.formatMessage({ description: 'Form Field label', defaultMessage: 'Environment Preset' })}
-          >
-            <Select
-              selectedOption={selectedOption}
-              onChange={(e) => {
-                if (e.detail.selectedOption.value === 'n/a') {
-                  setSceneProperty(KnownSceneProperty.EnvironmentPreset, undefined);
-                } else {
-                  setSceneProperty(KnownSceneProperty.EnvironmentPreset, e.detail.selectedOption.value);
-                }
-              }}
-              options={presetOptions}
-              selectedAriaLabel={intl.formatMessage({ defaultMessage: 'Selected', description: 'label' })}
-              disabled={presetOptions.length === 0}
-              placeholder={intl.formatMessage({
-                defaultMessage: 'Choose an environment',
-                description: 'choose environment placeholder',
-              })}
-              expandToViewport
-            />
-          </FormField>
-        </SpaceBetween>
+        <MotionIndicatorVisibilityToggle />
       </ExpandableInfoSection>
 
+      {isEditing && (
+        <ExpandableInfoSection
+          title={intl.formatMessage({ description: 'ExpandableInfoSection Title', defaultMessage: 'Scene Settings' })}
+          defaultExpanded={false}
+        >
+          <SpaceBetween size='s'>
+            <FormField
```

Comment:
Don't we need validations on the select button?

Reply:
the only change to this part is the indentation when I added the `isEditing` condition to render this. there should be no issue with this part since we've done tests before